frontend/src/scenes/insights/EmptyStates/EmptyStates.tsx

```diff
@@ -40,6 +42,28 @@ export function InsightEmptyState({
     )
 }
 
+function FastModeLink({ insightProps }: { insightProps: InsightLogicProps }): JSX.Element {
+    const { setSamplingPercentage } = useActions(samplingFilterLogic(insightProps))
+    const { suggestedSamplingPercentage } = useValues(samplingFilterLogic(insightProps))
+    return (
+        <Tooltip
+            title={`Fast mode enables ${suggestedSamplingPercentage}% sampling for all insights you refresh, speeding up the calculation of results.`}
+            placement="bottom"
+        >
+            <Link
+                onClick={() => {
+                    setSamplingPercentage(suggestedSamplingPercentage)
+                    posthog.capture('sampling_enabled_on_slow_query', {
+                        samplingPercentage: suggestedSamplingPercentage,
+                    })
+                }}
+            >
+                <ThunderboltFilled className="mt-1" /> Fast mode {`(${suggestedSamplingPercentage}%)`}
```

Comment:
"Fast mode" is what we call the lightning bolt switch in the insight header. (We're considering removing this in #16861, pending analysis of actual usage). This here is just "sampling"

Reply:
Updated! 

![image](https://github.com/PostHog/posthog/assets/18598166/f5f23027-2c2a-4287-a6d4-6119e335d2fc)